src/_content/posts/call-for-2025-volunteers.md

```diff
@@ -0,0 +1,319 @@
+
+### Pre-conference responsibilities
+
+- Select on-site video production crew
+- Select post-conference processing crew
+- Coordinate both of the above teams with the online platform
+- Work with the venue team to ensure high-speed wired internet access to the on-site video crew
+- Transfer pre-recorded talks to the post-conference processing crew and then the online conference provider
+- Procure a wide variety of dongles for presenters to hook their laptops up to the projectors
+- Procure projectors and screens if the venue doesn't already have them, including one for the green room so speakers can test their setup
+
+### At-conference responsibilities
+
+- Help on-site crew set up equipment and ensure streaming works
+- Get dongles to the right rooms as needed
+- Help keynote speakers get presentation mode working if needed
+- Answer lots and lots of questions
+
+## Online Team
+
+Size: 2-4
+
+### Pre-conference responsibilities
+
+- Select an online conference provider (work with the DEFNA board to do this)
+- Determine what activities or new approaches will be taken (maybe a Spanish or French watch-party, writing group, meet-and-greets, etc)
+- Prep the platform with content (schedule, speakers, FAQ, and other pages)
+- Make sure there's a plan or process (automated or manual) to get day of ticket purchases into the platform
+- Recruit online moderators/volunteers
+- Prepare/update the online volunteer handbook
+- Coordinate with AV chair to test the platform
+
+### At-conference responsibilities
+
+- Coordinate with online moderators
+- Engage with audience as able to
+- Help facilitate any other online activities
+
+## Swag Committee
+
+Size: 1-2
+
+The swag committee is responsible for everything speakers and attendees take home from the conference (aside from knowledge!).
+
+### Pre-conference responsibilities
+
+- Design and order attendee t-shirt (with sponsor logos)
+- Design and order swag bags (with sponsor logos)
+- Design and order speaker gifts
+- Design and order DCUS-branded swag (stickers/magnets/etc.)
+- Order masks and COVID tests (if needed)
+- Order lanyards, badges, and badge flags
+
+### At-conference responsibilities
+
+- Recruit volunteers to stuff swag bags on Sunday afternoon
+- Get T-shirts and swag bags to registration desk
+- Get speaker gifts to the program team for distribution
+
+## Volunteer Committee
+
+Size: 2
+
+The volunteer committee is responsible for cultivating new volunteers, which can become organizers in future years.
+
+### Pre-conference responsibilities
+
+- Determine what roles need on-site volunteer support (session chairs, session managers, registration desk, etc.)
+- Set up sign-up for on-site volunteers and online volunteers (moderators/cheerleaders)
+- Draft announcements for calls for volunteers
```

Comment:
- Draft announcements for calls for volunteers **and handle communication with volunteers**